src/renderer/widgets/AddProxyModal/model/form-model.ts

```diff
@@ -55,19 +71,19 @@ const feeChanged = createEvent<string>();
 const isFeeLoadingChanged = createEvent<boolean>();
 const isProxyDepositLoadingChanged = createEvent<boolean>();
 
-const $fee = createStore<string>('0');
-const $oldProxyDeposit = createStore<string>('0');
-const $newProxyDeposit = createStore<string>('0');
-const $multisigDeposit = createStore<string>('0');
-const $isFeeLoading = createStore<boolean>(true);
-const $isProxyDepositLoading = createStore<boolean>(true);
+const $fee = createStore<string>('0').reset(formSubmitted);
```

Comment:
What if I return from Confirm to Form screen? Will I be able to see all the recent data?